# In file included from /path/to/filters/sheets/excel/import/ExcelImport.cpp:1443:
# /path/to/build/filters/sheets/excel/import/calligra_filter_xls2ods_autogen/include/ExcelImport.moc:57:6: error: no member named 'ExcelImportFactory' in namespace 'Swinder'; did you mean simply 'ExcelImportFactory'?
# void Swinder::ExcelImportFactory::qt_static_metacall(QObject *_o, QMetaObject::Call _c, int _id, void **_a)
 

diff --git a/filters/sheets/excel/import/ExcelImport.cpp b/filters/sheets/excel/import/ExcelImport.cpp
index 2c28d5a05cca2baa53189874510fc5d36b25e7fb..6407095a27e1cc2a4e46a7b0f7936d2c4bf8dbf8 100644
--- a/filters/sheets/excel/import/ExcelImport.cpp
+++ b/filters/sheets/excel/import/ExcelImport.cpp
@@ -81,6 +81,9 @@
 #include "ImportUtils.h"
 #include "conditionals.h"
 
+using namespace Swinder;
+using namespace XlsUtils;
+
 // Enable this definition to make the filter output to an ods file instead of
 // using m_chain.outputDocument() to write the spreadsheet to.
 //#define OUTPUT_AS_ODS_FILE
@@ -91,9 +94,6 @@ static const qreal SIDEWINDERPROGRESS = 40.0;
 static const qreal ODFPROGRESS = 40.0;
 static const qreal EMBEDDEDPROGRESS = 15.0;
 
-using namespace Swinder;
-using namespace XlsUtils;
-
 static qreal offset( unsigned long dimension, unsigned long offset, qreal factor ) {
     return (float)dimension * (float)offset / factor;
 }
